client/pom.xml

```diff
@@ -50,5 +50,58 @@
 			<artifactId>jzlib</artifactId>
 			<version>1.1.3</version>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-server</artifactId>
+			<version>9.2.14.v20151106</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.github.tomakehurst</groupId>
+			<artifactId>wiremock</artifactId>
+			<version>1.57</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.mortbay.jetty</groupId>
+					<artifactId>jetty</artifactId>
+				</exclusion>
```

Comment:
Are you sure this works?! I'd be very surprised. Wiremock 1.5.7 is built on top of very old Jetty (different groupId and package name). It seems that upgrade is in progress, but Wiremock 2.0.8-beta still targets Jetty 9.2, which is still sadly one generation behind, and AFAIK not exactly compatible. It would be great if Wiremock could finally target latest Jetty.